pages/learn/manage/account.md

```diff
@@ -8,6 +8,44 @@ title: Account management
 
 {{> "general/signUp"}}
 
+## Access tokens
+
+Access tokens are used for authentication in the resin.io [API][api], [CLI][cli], and [Node.js][node-sdk] and [Python][python-sdk] SDKs. They are managed in the *Access tokens* tab of the [*Preferences*][prefs] page, which can be found via the dropdown menu in the upper-right corner of the dashboard:
+
+<img src="/img/common/preferences/access_tokens.png" width="80%">
+
+There are two types of access tokens: session tokens and API keys. Both authentication types provide user-level permissions, meaning any user or application with one of these tokens can make changes across devices, applications, and the user account.
+
+### Session tokens
+
+Session tokens are retrieved from the *Preferences* page, and they can be refreshed with the [API][api-refresh]. These tokens expire after seven days, and they cannot be revoked.
+
+### API keys
+
+API keys are named tokens that do not expire and can be revoked as needed. To create a new API key, make sure you are in the *Access tokens* tab of the *Preferences* page, then select *Create API key*:
+
+<img src="/img/common/preferences/create_api_key.png" width="80%">
+
+You'll see a required field for *Token name*, as well as an optional field for *Token description*:
```

Comment:
Do we have any requirements around Token name that are worth documenting (i.e. length, whitespace, special chars)?

Reply:
Good question for @thgreasi

Reply:
I'm naming them based on the scope that they will be used. eg for the CLI, for devices of application X.
Other than these, I can't think of any other suggestion. Maybe we could check what GH docs mention for their API keys.